packages/trpc/server/middlewares/sessionMiddleware.ts

```diff
@@ -73,7 +73,7 @@ export async function getUserFromSession(ctx: TRPCContextInner, session: Maybe<S
 
   return {
     ...user,
-    avatar: `${WEBAPP_URL}/${user.username}/avatar.png?${organization.id}` && `orgId=${organization.id}`,
+    avatar: `${WEBAPP_URL}/${user.username}/avatar.png${organization.id ? `?orgId=${organization.id}` : ""}`,
```

Comment:
This seems wrong

--
Edit: nvm, it is correct

Reply:
We shouldn't use this URL anymore though, but out of scope. (+ an issue with orgId which is probably why it's like this)